aten/src/ATen/native/Normalization.cpp

```diff
@@ -466,6 +468,15 @@ Tensor layer_norm(const Tensor& input, IntArrayRef normalized_shape,
     const Tensor& weight /* optional */, const Tensor& bias /* optional */,
     double eps, bool cudnn_enabled) {
 
+    if (!input.is_cuda()) {
```

Comment:
input.device().is_cpu() as we are also support HIP and potentially other tensors

Reply:
Thanks for the advice!